Jenkinsfile

```diff
@@ -204,10 +204,22 @@ pipeline {
               }
             }
             failure {
-              slackSend channel: "#${params.SLACK_TO}",
-                        message: ":x: JOB FAILURE: $JOB_NAME - $BUILD_ID\n\n${BUILD_URL}console\n\n<!channel>",
-                        teamDomain: "ihme",
-                        tokenCredentialId: "slack"
+              script {
+                if (${GIT_BRANCH} == "main") {
+                  echo 'This is the main branch. Sending a failure message to Slack.'
+                  slackSend channel: "#${params.SLACK_TO}",
+                            message: ":x: JOB FAILURE: $JOB_NAME - $BUILD_ID\n\n${BUILD_URL}console\n\n<!channel>",
+                            teamDomain: "ihme",
+                            tokenCredentialId: "slack"
+                } else {
+                  echo "This is branch ${GIT_BRANCH}. Sending a failure message to Slack."
+                  // TODO: DM the developer instead of the slack channel
+                  slackSend channel: "#simsci-ci-status-test",
+                            message: ":x: JOB FAILURE: $JOB_NAME - $BUILD_ID\n\n${BUILD_URL}console",
+                            teamDomain: "ihme",
+                            tokenCredentialId: "slack"
+                }
+              }
```

Comment:
Aren't these blocks identical other than the name of the channel? Can't you do
```groovy
if (${GIT_BRANCH} == "main") {
    channelName = "#${params.SLACK_TO}"
} else {
    channelName = "#simsci-ci-status-test"
}
echo ...
slackSend channel: channelName,
    ...
```

Reply:
Made this change.